test/utils/redis/redisMock.js

```diff
@@ -0,0 +1,19 @@
+const RedisClientMock = class {
+	get(...args) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['args' is defined but never used. Allowed unused args must match /app|req|res|next|options|params/. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4141092)